makiflow/models/segmentation/utils.py

```diff
@@ -67,7 +67,11 @@ def merging_masks(masks, index_of_main_mask, priority):
             continue
 
         for prior in priority:
-            main_mask[masks[i] == prior * boolean_mask] = prior
+            # Take indexes there `prior` class in `masks[i]` and multiply on our `boolean_mask` to remove used indexes
```

Comment:
The comment doesn't make sense. You can write an entire paragraph, but make it clear.